bin/runtime/src/lib.rs

```diff
@@ -327,14 +327,16 @@ impl pallet_aleph::Config for Runtime {
 }
 
 parameter_types! {
-    // We allow 10kB keys. This is in 100% blind guess.
+    // We allow 10kB keys, proofs and public inputs. This is in 100% blind guess.
     pub const MaximumVerificationKeyLength: u32 = 10_000;
+    pub const MaximumDataLength: u32 = 10_000;
```

Comment:
while this could make sense in general, this whole file has _a lot_ of such parameters, therefore for consistency I would leave it as it is now, especially that these two parameters are used (only) right below declaration